### DISCUSSION TITLE
NewDefaultServer replacement

### DISCUSSION THREAD

**Discussion by idc77** [99designs/gqlgen#3447] [Q&A] [answered]
https://github.com/99designs/gqlgen/pull/3404 says it's not intended for prod use and that people have found out the hard way.
I'm using it in prod like that so what should I replace it with?

**Comment by StevenACoffman** [comment, marked as answer]
@idc77 Hey, so you can feel free to copy-paste NewDefaultServer into your own project and then modify it the way the comment for NewDefaultServer suggests. Add or remove the transports to match your needs, and customize the transport configs.

For example, if you don't need websockets, don't add the websocket transport. If you do need websocket transport, customize the settings to match your workloads.
